crates/wadm/src/server/handlers.rs

```diff
@@ -750,19 +764,25 @@ impl<P: Publisher> Handler<P> {
                 .map(|_| DeployModelResponse {
                     result: DeployResult::Acknowledged,
                     message: format!("Successfully undeployed application {name}"),
+                    name: name.to_string(),
+                    version: deployed_version.clone().unwrap_or_default(),
```

Comment:
Isn't this potentially a bit misleading to send back a version when you've undeployed something, because by definition it is not in fact deployed.

Perhaps it's splitting hairs over semantics though and it would. be better to always send back a version regardless.

Reply:
Potentially, yes. I could set the version to an option to account for this 🤔

Reply:
@joonas changed to an option!